src/Invariants/Equality.jl

```diff
@@ -0,0 +1,56 @@
+
+"""
+    ==(
+        a::AbstractOptimalEdgeSet,
+        b::AbstractOptimalEdgeSet
+    )::Bool
+
+Determine if two sets of edges are equivalent in an undirected graph context.
+
+This method treats each edge as undirected, meaning the order of vertices in
+each edge tuple or `Graphs.SimpleGraphs.SimpleEdge` is ignored. Two sets of edges
+are considered equal if they contain the same pairs of vertices, regardless of
+the order in which the vertices appear in each edge.
+
+# Arguments
+- `a::AbstractOptimalEdgeSet`: The first set of edges to compare.
+- `b::AbstractOptimalEdgeSet`: The second set of edges to compare.
+
+# Returns
+- `true` if both sets of edges represent the same undirected connections, `false` otherwise.
+
+# Examples
+```julia
+julia> using Graphs.SimpleGraphs: SimpleEdge
+
+julia> using GraphProperties.Invariants
+
+julia> a = OptimalEdgeSet([SimpleEdge(1, 2), SimpleEdge(2, 3)])
+
+julia> b = OptimalEdgeSet([SimpleEdge(2, 1), SimpleEdge(3, 2)])
+
+julia> a == b
+true
+```
+"""
+
+using Graphs
```

Comment:
Add this to `Invariants.jl` (see # this below):

```julia
import Base: == # this 

using Colors
using Combinatorics
using Graphs
using Graphs.SimpleGraphs: SimpleEdge # this 
```

And then remove these from `Equality.jl`. Also, change the name to `equality.jl`